package.json

```
@@ -65,14 +65,14 @@
     "@types/fs-extra": "^9.0.11",
     "@types/js-yaml": "^4.0.1",
     "@types/lodash": "^4.14.170",
-    "@types/mocha": "^8.2.2",
+    "@types/mocha": "^10.0.1",
     "@types/node": "^15.12.2",
     "@typescript-eslint/eslint-plugin": "^4.26.1",
     "@typescript-eslint/parser": "^4.26.1",
     "chai": "^4.3.4",
-    "eslint": "^7.28.0",
-    "eslint-config-airbnb-base": "^13.2.0",
-    "eslint-plugin-import": "^2.23.3",
+    "eslint": "^7.32.0 || ^8.2.0",
```

I wonder if we should simply upgrade to 

```suggestion
    "eslint": "^8.2.0",
```

Or is it necessary to support both v7 and v8? I don't think it is, just because `yarn.lock` will lock us into a particular version that presumably only matches one of these specifiers.

---

Fair enough! I can try this version and make sure it doesn't fail tests

---

Changing just to `"eslint": "^8.2.0",` has peer dependency failures.

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @typescript-eslint/eslint-plugin@4.26.1
npm ERR! Found: eslint@8.47.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^8.2.0" from the root project
npm ERR!   peer eslint@"^6.0.0 || ^7.0.0 || >=8.0.0" from @eslint-community/eslint-utils@4.4.0
npm ERR!   node_modules/@eslint-community/eslint-utils
npm ERR!     @eslint-community/eslint-utils@"^4.2.0" from eslint@8.47.0
npm ERR!   4 more (@typescript-eslint/experimental-utils, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^5.0.0 || ^6.0.0 || ^7.0.0" from @typescript-eslint/eslint-plugin@4.26.1
npm ERR! node_modules/@typescript-eslint/eslint-plugin
npm ERR!   dev @typescript-eslint/eslint-plugin@"^4.26.1" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: eslint@7.32.0
npm ERR! node_modules/eslint
npm ERR!   peer eslint@"^5.0.0 || ^6.0.0 || ^7.0.0" from @typescript-eslint/eslint-plugin@4.26.1
npm ERR!   node_modules/@typescript-eslint/eslint-plugin
npm ERR!     dev @typescript-eslint/eslint-plugin@"^4.26.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

---

@tbroadley - I think it may make sense (or be less noisy) to support both for now?

---

Going back to `"eslint": "^7.32.0 || ^8.2.0",` allowed me to `npm install` happily.

---

I'll resolve this for now.